pkg/ccl/backupccl/restore_job.go

```diff
@@ -433,16 +433,7 @@ func WriteDescriptors(
 			}
 			return err
 		}
-
-		bdg := catalogkv.NewOneLevelUncachedDescGetter(txn, codec)
-		descs := make([]catalog.Descriptor, 0, len(databases)+len(tables))
-		for _, table := range tables {
-			descs = append(descs, table)
-		}
-		for _, db := range databases {
-			descs = append(descs, db)
-		}
-		return catalog.ValidateSelfAndCrossReferences(ctx, bdg, descs...)
+		return nil
```

Comment:
I looked into this in detail while chasing an unrelated bug and realized that there's no point to these checks here because the validation-on-write checks kick in very soon after.